Packs/ApiModules/Scripts/CoreXQLApiModule/CoreXQLApiModule.py

```diff
@@ -641,6 +647,18 @@ def start_xql_query_polling_command(client: CoreClient, args: dict) -> Union[Com
     if not args.get('query_name'):
         raise DemistoException('Please provide a query name')
     execution_id = start_xql_query(client, args)
+    if execution_id == 'FAILURE':
+        # the 'start_xql_query' function failed because it reached the maximum allowed number of parallel running queries.
+        # running the command again using polling with an interval of 'interval_in_secs' seconds.
+        command_results = CommandResults()
+        interval_in_secs = int(args.get('interval_in_seconds', 10))
+        scheduled_command = ScheduledCommand(command='xdr-xql-generic-query', next_run_in_seconds=interval_in_secs,
+                                             args=args, timeout_in_seconds=600)
```

Comment:
```suggestion
                                             args=args, timeout_in_seconds=200)
```
Lets start with a lower value for timeout

Reply:
maybe 180 / 240? to align with the minute cycles